src/osd/PrimaryLogPG.cc

```diff
@@ -2244,6 +2256,7 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
 	get_osdmap()->require_osd_release >= ceph_release_t::kraken) {
       record_write_error(op, oid, nullptr, r);
     } else {
+      //TODO: jaeger: add span 
```

Comment:
don't think we need a span for returning an error to the client

Reply:
was not sure whether it would be useful, ack :+1: